packages/api/src/controllers/sensorController.js

```diff
@@ -256,6 +338,29 @@ const sensorController = {
       });
     }
   },
+
+  async retireSensor(req, res) {
+    try {
+      const { external_id, location_id, farm_id, partner_id } = req.body.sensorInfo;
+      const user_id = req.user.user_id;
+      const { access_token } = await IntegratingPartnersModel.getAccessAndRefreshTokens(
+        'Ensemble Scientific',
+      );
+      const external_integrations_response = await FarmExternalIntegrationsModel.getOrganizationId(
+        farm_id,
+        partner_id,
+      );
+      const org_id = external_integrations_response[0].organization_uuid;
```

Comment:
If you look at my comment on `farmExternalIntegrationsModel.js` you can update this to be:
```suggestion
      const org_id = external_integrations_response.organization_uuid;
```